components/Chat/Message/message-context-menu/message-context-menu-utils.tsx

```
@@ -0,0 +1,90 @@
+  const isAttachment =
+    isRemoteAttachmentMessage(message) || isStaticAttachmentMessage(message);
+  const isTransaction = isTransactionReferenceMessage(message);
+
+  if (!isAttachment && !isTransaction) {
+    items.push({
+      title: translate("copy"),
+      rightView: <TableViewPicto symbol="doc.on.doc" />,
+      id: CONTEXT_MENU_ACTIONS.COPY_MESSAGE,
+      action: () => {
+        const messageStringContent = getMessageStringContent(message);
+        if (!!messageStringContent) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)